spec/services/ncr/reporter_spec.rb

```diff
@@ -21,40 +23,27 @@
     end
   end
 
-  describe ".proposals_tier_one_pending" do
+  describe "#proposals_tier_one_pending" do
     it "only returns Proposals where Tier One approval is actionable" do
```

Comment:
I reduced these where I thought I could leverage the `@work_order` and `@completed_work_order`. I might have removed too much...

But as far as I understand, there are two real work orders in play here, and the two auto-created ones work for this test. ?